database/migrations/sqls/20240722104637-created-new-tables-up.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE bid (
+    bid_id SERIAL PRIMARY KEY,
+    tender_id INTEGER,
+    bidder_id INTEGER,
+    buyer_id INTEGER,
+    bidding_date DATE,
+    status VARCHAR(100),
+    bidding_amount DECIMAL(15, 2) CHECK (bidding_amount >= 0),
+    cover_letter VARCHAR(255),
+    suggested_duration INTEGER,
+    FOREIGN KEY (tender_id) REFERENCES tender (tender_id),
+    FOREIGN KEY (bidder_id) REFERENCES bidder (user_id),
+    FOREIGN KEY (buyer_id) REFERENCES buyer (user_id)
+);
```

Comment:
shall write DROP table command at first in this file?

Reply:
Actually, instead of dropping it, you can rename it with `ALTER` and then use other `ALTER` statements to make the modifications you want to make.

Reply:
Just for clearification. Right now table has name bid. You want me to change it? Because I created that in new migration file, It wasn't existed before.

Reply:
I apologize. I thought there was already a table called bids. Dismiss this comment. What you are doing is fine.